terraform/app/modules/cloudfront/variables.tf

```diff
@@ -38,3 +38,15 @@ variable default_header_list {
   ]
 }
 
+variable route53_zones {
+  default = {
+    "teaching-jobs.service.gov.uk"      = "Z15RHKLIWUJQ8O"
+    "teaching-vacancies.service.gov.uk" = "Z1QCHCNGLRJMK0"
+  }
+}
+
+locals {
+  is_production             = length(regexall(".*production.*", var.environment)) > 0
+  cloudfront_route53_prefix = local.is_production ? "www" : var.environment
+  cloudfront_route53_zones  = local.is_production ? var.route53_zones : {}
```

Comment:
if cloudfront_route53_zones is empty, it won't create the DNS records?

Reply:
Renamed the variables for clarity: `local.`:
- `route53_zones`
- `route53_zones_with_a_records`
- `route53_zones_with_cnames`

CNAMEs are created when a distribution is defined.
If the environment is production, A records are additionally defined.